package.json

```diff
@@ -22,12 +22,14 @@
   "devDependencies": {
     "@cloudflare/workers-types": "^4.20221111.1",
     "@types/jest": "^29.2.5",
+    "@types/jsonwebtoken": "^9.0.2",
     "@types/node": "^18.11.18",
     "@types/node-fetch": "^2.6.2",
     "@typescript-eslint/eslint-plugin": "^5.47.1",
     "@typescript-eslint/parser": "^5.47.1",
     "eslint": "^8.31.0",
     "jest": "^29.3.1",
+    "jsonwebtoken": "^9.0.1",
```

Comment:
Why are we installing the jsonwebToken package?

Reply:
to mock cloudfareworker jwt